macros/models/_sector/nft/nft_transfers.sql

```diff
@@ -25,13 +25,9 @@ FROM(
     , et."from" AS executed_by
     {%- endif %}
     , t.evt_tx_hash AS tx_hash
-    , '{{blockchain}}' || cast(t.evt_tx_hash as varchar) || '-{{token_standard_721}}-' || cast(t.contract_address as varchar) || '-' || cast(t.tokenId as varchar) || '-' || cast(t."from" as varchar) || '-' || cast(t.to as varchar) || '-1-' || cast(t.evt_index as varchar) AS unique_transfer_id
+    , {{ dbt_utils.generate_surrogate_key(['t.evt_tx_hash', 't.evt_index', 't.tokenId', 1]) }} as unique_transfer_id -- For backward compatibility
```

Comment:
any reason this macro passes in `1` at the end, while the others don't?

Reply:
`amount` is hardcoded to `1` above.

Reply:
ahh, got it, thank you